recipe/meta.yaml

```diff
@@ -1,49 +1,62 @@
 {% set version = "3.18.0" %}
 
-
 package:
     name: freeimage
     version: {{ version }}
-    # note: also update version in run_test.py
+    # note: also update version in CMakeLists and in tests.
 
 source:
-    fn: FreeImage{{ version.replace(".", "") }}.zip
-    url: http://downloads.sourceforge.net/freeimage/FreeImage{{ version.replace(".", "") }}.zip
+  - url: http://downloads.sourceforge.net/freeimage/FreeImage3180.zip
     sha256: f41379682f9ada94ea7b34fe86bf9ee00935a3147be41b6569c9605a53e438fd
     patches:
-        - Makefile.osx.patch  # [osx]
+        - Makefile.osx.patch  # [osx] 
 
 build:
-    number: 0
-    skip: true  # [win and vc<14]
-    binary_relocation: False  # [osx]
-    run_exports:
-        - {{ pin_subpackage('freeimage', max_pin='x.x') }}
+  number: 1
+  skip: true  # [win or osx]
```

Comment:
Can you enable OSX? I don't think we can merge with OSX disabled.

Reply:
why can we not?
I had issues with libpng and I currently don't have a OS X to test locally. (I think the patch you pointed me at will not fix it).
But I'll attempt a build let's see.

Reply:
ok doesn't build . 

can we merge like this for now until i have a mac mini?

Reply:
cc @conda-forge/freeimage

Reply:
You can use the azure to test builds. We have enough capacity that submitting commits to test things is totally fine. You don't need your own mac to test osx.